sourcemash/frontend/static/js/views/sidenav_view.js

```diff
@@ -61,13 +71,18 @@ Sourcemash.Views.SidenavView = Backbone.View.extend({
     });
   },
 
+  refreshPage: function(e){
+    e.preventDefault();
+    window.location.reload(true);
```

Comment:
Every iteration of this I've been playing with hasn't been successful. Not sure what to do at this point. I think we should just bite the bullet on the reset, knowing that it actually works correctly and nothing in the reset-and-fetch variety seems to.